src/main/resources/com/tikal/hudson/plugins/notification/HudsonNotificationProperty/config.jelly

```diff
@@ -4,98 +4,67 @@
 
     <f:section title="Job Notifications">
         <f:entry title="Notification Endpoints" field="endpoints">
-            <f:repeatable name="endpoints" var="endpoint"
-                items="${instance.endpoints}" add="${%Add Endpoint}">
-                <table class="center-align">
```

Comment:
there's migration guides here: https://www.jenkins.io/doc/developer/views/table-to-div-migration/

basically:

table => div
tr => div class="tr"
td => div

and keep the rest of the markup the same (not strictly needed if you're dropping support for old core, but normally easier to stay compatible if you don't change the markup too)

you'll want to use a local taglib so that the right tags will be used where required

here's a full example doing it for table:
https://github.com/jenkinsci/git-plugin/pull/973/files

duplicate the tag pretty much for tr and td, we normally call it rowWrapper and entryWrapper

also you're IDE has changed all  the tabs to spaces would be good if that could be avoided

here's another example where they are extracted into a standalone taglib:
https://github.com/jenkinsci/postbuild-task-plugin/pull/6

Reply:
Ah cool I will take a look, I was just experimenting with getting our work install running properly, I will make these changes in the morning